packages/graphiql-react/src/editor/common.ts

```diff
@@ -0,0 +1,53 @@
+let isMacOs = false;
+
+if (typeof window === 'object') {
+  isMacOs = window.navigator.platform === 'MacIntel';
```

Comment:
The current M1's seem to return `macOS` here for Brave browser, `MacIntel` for Chrome, `MacIntel` for FireFox and `MacIntel` we could alternatively do something like `window.navigator.platform.toLowerCase().startsWith('mac')`

Reply:
Interesting, when I run this in Brave on my M1 Max I get `MacIntel` 🤔 However, I don't see anything that speaks against making this more loose 👍

Reply:
Heh, could've been me using `navigator.userAgentData.platform` as well 😅 apologies